edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test.csproj

```diff
@@ -16,10 +16,10 @@
 
   <ItemGroup>
     <PackageReference Include="Docker.DotNet" Version="3.125.2" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.3.0" />
```

Comment:
why do you need to update it? we should keep 1 version of same NuGet package across projects.

Reply:
Should I update everywhere?

Reply:
downgraded to 15.9.0